src/grib_accessor_class_data_ccsds_packing.cc

```diff
@@ -234,14 +233,20 @@ static int pack_double(grib_accessor* a, const double* val, size_t* len)
         return err;
     if ((err = grib_get_long_internal(hand, self->decimal_scale_factor, &decimal_scale_factor)) != GRIB_SUCCESS)
         return err;
-
     if ((err = grib_get_long_internal(hand, self->ccsds_flags, &ccsds_flags)) != GRIB_SUCCESS)
         return err;
     if ((err = grib_get_long_internal(hand, self->ccsds_block_size, &ccsds_block_size)) != GRIB_SUCCESS)
         return err;
     if ((err = grib_get_long_internal(hand, self->ccsds_rsi, &ccsds_rsi)) != GRIB_SUCCESS)
         return err;
 
+    // ECC-1602: Performance improvement
+    ccsds_flags &= ~AEC_DATA_MSB;  // enable little-endian
+    ccsds_flags &= ~AEC_DATA_3BYTE;  // disable support for 3-bytes per value
+    unsigned short is_little_endian = 1;
```

Comment:
The include file /usr/local/apps/aec/1.0.6/include/libaec.h states:

Samples are stored with their most significant bit first. **This has nothing to do with the endianness of the host**. Default is LSB.

#define AEC_DATA_MSB 4

Reply:
I don't know why the flag is called like that, but the libaec documentation says the following: "AEC_DATA_MSB: input data is stored most significant byte first i.e. big endian. Default is little endian on all architectures."

Reply:
I forgot to mention that the default value for AEC_DATA_MSB=true is stored in the definition files. Since we load this flag from a definition file, we must reset it to 0 to use libaec on a little-endian machine like ATOS HPC efficiently. In my opinion, storing machine-dependent flags like AEC_DATA_MSB and AEC_DATA_3BYTE in the definition files is unnecessary.